tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -888,5 +888,39 @@ public void GDIPlusLibrary_IsPresentInTheImage(string tagName)
                 },
                 result.GetDebugInfo());
         }
+
+        [Theory]
+        [InlineData("lts-versions", "3")]
+        [InlineData("vso", "2")]
+        [InlineData("vso-focal", "3")]
+        [InlineData("latest", "2")]
+        [InlineData("latest", "3")]
+        public void JamSpell_CanBe_InstalledInTheBuildImage(string tagName, string pythonVersion)
```

Comment:
Should we put this test in `PythonSampleAppsTests.cs` instead

Reply:
good catch .. done !